packages/dialtone-vue2/components/avatar/avatar.test.js

```diff
@@ -9,6 +9,7 @@ const MOCK_INITIALS = 'JN';
 const MOCK_SIZE = 'lg';
 const MOCK_GROUP = 25;
 const MOCK_CUSTOM_CLASS = 'my-custom-class';
+const MOCK_ICON_SLOT = '<template #icon=>Icon</template>';
```

Comment:
You can actually mount a component here, check [split button tests](https://github.com/dialpad/dialtone/blob/staging/packages/dialtone-vue2/components/split_button/split_button.test.js#L190).

Also I think that template is missing the quotation marks ("") right?

You can do something like

```suggestion
const MOCK_ICON_SLOT = '<dt-icon-user />';
```